unit-tests/Extension/ExceptionsTest.php

```diff
@@ -143,4 +144,37 @@ public function testExceptionRethrow()
             $this->assertSame(8, $e->getLine());
```

Comment:
Make sense… Your test patch added 3 lines to the top of test/exceptions.zep this is why the number is 11 now